sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

```diff
@@ -10,24 +11,31 @@ internal actual object SentryBridge {
         return SentryId(androidSentryId.toString())
     }
 
+    actual fun captureMessage(message: String, scopeCallback: SentryScopeCallback): SentryId {
```

Comment:
`scopeCallback` can be an optional parameter with a default value, you dont need 2 overloads.

Reply:
Kotlin -> Obj-C is sometimes ugly so default parameters are pretty much nonexistent

https://github.com/getsentry/sentry-kotlin-multiplatform/blob/main/sentry-samples/kmp-app/iosApp/iosApp/ContentView.swift#L15

iOS users would need to write it like this: `captureMessage("message: "...", scopeCallback = nil)